packages/office-ui-fabric-react/src/components/Button/CommandButton/CommandButton.scss

```diff
@@ -9,61 +9,61 @@
 
 $button-commandButtonHeight: 40px;
 
-.ms-Button--command {
-  @include text-align(left);
+.ms-Button {
+  &.ms-Button--command {
```

Comment:
Is this specificity necessary? We usually try to avoid combining the modifier with the base class like `.ms-Button.ms-Button--command`